test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/LegacyPackageReferenceRestoreUtilityTests.cs

```diff
@@ -150,7 +150,7 @@ public async void DependencyGraphRestoreUtility_LegacyPackageRef_Restore_Generat
 
                     // set up projects
                     var projectTargetFrameworkStr = "net45";
-                    var fullProjectPathB = Path.Combine(testSolutionManager.TestDirectory, "ProjectB", "project2.csproj");
+                    var fullProjectPathB = Path.Combine(testSolutionManager.TestDirectory, "ProjectB", "Project2.csproj");
```

Comment:
why change this?

Reply:
extra test to ensure that the lock file lower cases project names.